src/Factory/Cache/DoctrineCacheFactory.php

```diff
@@ -14,29 +14,39 @@ class DoctrineCacheFactory implements FactoryInterface
      *
      * @return Cache
      */
-    public function create(ContainerInterface $serviceLocator)
+    public function create(ServiceLocatorInterface $serviceLocator)
     {
+        /** @var Cache */
         return $serviceLocator->get($this->getConfiguredServiceName($serviceLocator));
     }
 
     /**
      * @param ServiceLocatorInterface $serviceLocator
      *
-     * @return Cache
+     * @return string
      */
-    private function getConfiguredServiceName(ContainerInterface $serviceLocator)
+    private function getConfiguredServiceName(ServiceLocatorInterface $serviceLocator)
```

Comment:
Nicking @faggc 's comment on the feature toggle PR, is there any reason for the `@return` comment syntax here, rather than putting `:string` on the end of the function call? It would be consistent with the invoke function below.

Reply:
added : string